twitter-eventlistener-plugin/src/main/java/com/twitter/presto/plugin/eventlistener/QueryCompletedEventScriber.java

```diff
@@ -55,6 +61,23 @@ public void handle(QueryCompletedEvent event)
     }
   }
 
+  private static long getLongOrZero(String strVal)
+  {
+    try {
+      return DataSize.valueOf(strVal).toBytes();
+    }
+    catch (IllegalArgumentException e) {
+      log.warn(e, "Failed to parse io.airlift.units.DataSize string, returning 0");
```

Comment:
can you include strVal and info about the query here? Without that, this log entry will be in-actionable.